crypto/ec/ec_key.c

```diff
@@ -11,6 +11,7 @@
 #include "internal/cryptlib.h"
 #include <string.h>
```

Comment:
In the context of previous side note. I have some wage memory that it, header ordering rule, was indirectly discussed at some point. It was formulated as "cryptlib.h over everything else", and we do see instance of it here. Do note that cryptlib.h did include e_os.h till now...